ProjectShop/order/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.contrib.auth import get_user_model
+from UserApp.models import User
+
+
+# Create your models here.
+class Order(models.Model):
+    order_date = models.DateTimeField(auto_now_add=True)
+    PAYMENT_METHODS = (
+        (1, "Card"),
+        (2, "Cash")
+    )
+    payment_method = models.IntegerField(verbose_name="Payment Method", choices=PAYMENT_METHODS, default=2)
```

Comment:
I propose to add translation:
from django.utils.translation import ugettext_lazy as _

Reply:
I propose to use enum for choices https://docs.djangoproject.com/en/3.2/ref/models/fields/#field-choices-enum-types